src/lib.rs

```diff
@@ -73,6 +73,7 @@ extern crate raft;
 extern crate rand;
 extern crate regex;
 extern crate rocksdb;
+extern crate rustc_serialize;
```

Comment:
Better not use an internal crate.

Reply:
rustc_serialize is not an internal crate,  see https://github.com/tikv/tikv/blob/d642e43273850100ea6d37e2c292d164a0b0f355/Cargo.toml#L70